.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: dart
 dart:
   - dev
-  - stable
```

Comment:
Does this really need to drop stable? I don't see BigInt being used...

Reply:
The except in the test returns different results on stable and dev, cause of difference in int64 behavior in the VM. Fixed it for dev16, but now it fails on stable.

Reply:
Oh...hrm...

Couldn't we modify the tests so they work on both?

My concern: if we drop testing for stable, we should likely drop it in the pubspec, too.

But it'd be nice to avoid that since – AFAICT – there is no behavior change

Reply:
With dev 16, min-1=max and max+1=min for int 64, so it wraps around.  So now min-1 and max+1 are contained in the buffer, whereas previously they were not. This is a corner case, but it is a change in behavior. So change the pubspec?

Reply:
Sure, change the pubspec to require the next version of the SDK,

Reply:
@kevmoo ptal.